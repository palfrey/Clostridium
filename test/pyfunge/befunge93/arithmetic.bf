72+.72-.72*.72/.56789/+-*.@
