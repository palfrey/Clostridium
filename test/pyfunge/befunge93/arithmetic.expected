9 5 14 3 -5 
